docs/dev/setup.md

```diff
@@ -0,0 +1,430 @@
+
+For Windows users only [Docker ToolBox](https://docs.docker.com/toolbox/toolbox_install_windows/) works. The reason is that Docker Toolbox uses VirtualBox for virtualization and [Docker for Windows](https://docs.docker.com/docker-for-windows/install/#about-windows-containers) uses the native OS virtualization (Hyper-V). Since PCF Dev requires VirtualBox and using VirtualBox on Windows requires disabling Hyper-V, it is currently not an option to use Docker for Windows.
+
+### Setup VirtualBox
+
+> **Note:** For Windows make sure Hyper-V is turned off.
+
+If you decide to use Docker ToolBox, VirtualBox is installed as part of it.
+
+If you are a Mac user and you decide to use Docker for Mac, you would need to seperatly install VirtualBox. You may download it from [here](https://www.virtualbox.org/wiki/Downloads).
+
+### Setup PCF Dev
+
+The [following guide](https://pivotal.io/platform/pcf-tutorials/getting-started-with-pivotal-cloud-foundry-dev/introduction) describes how to install PCF Dev.
+
+The Service Manager components use go1.10 and currenty  the buildpack installed in PCF Dev does not support go1.10. The following steps install the latest go buildpack version on PCF Dev:
+
+* Download the latest Release from [here](https://github.com/cloudfoundry/go-buildpack/releases)
+* Rename the zip so it cotnains no dots (.) and slashes (-)
+* Navigate to the directory the zip was downloaded to and run:
+
+    ```bash
+    cf create-buildpack go_buildpack2 <buildpackzipname>.zip 1
+    ```
+
+### Setup Minikube
+
+#### Install Minikube
+
+In general the installation steps are in the [Installation section of the getting started guide](https://kubernetes.io/docs/getting-started-guides/minikube/#installation).
+
+Alternatively, Windows users may follow these steps:
+
+> **Note:** On Windows Minikube versions 26.x does not work on Windows. Currently version 25.2 works properly.
+
+* Setup chocolately package manager
+
+    ```powershell
+    @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+    ```
+
+* install kubectl
+
+    ```bat
+    choco install kubernetes-cli
+    ```
+
+* install minikube
+
+    ```bat
+    choco install minikube --version 0.25.2
+    ```
+
+#### Start and Prepare Minikube
+
+* install Helm
+
+    In general Helm installation is explained [here](https://github.com/kubernetes/helm/blob/master/docs/install.md).
+
+    Alternatively, Windows users may follow [this guide](https://medium.com/@JockDaRock/take-the-helm-with-kubernetes-on-windows-c2cd4373104b).
+
+* start minikube with RBAC
+
+    ```bash
+    minikube start --extra-config=apiserver.Authorization.Mode=RBAC

```

Comment:
minikube start --bootstrapper localkube --extra-config=apiserver.Authorization.Mode=RBAC

As there have some issues with minikube and it fails to start on Mac (https://github.com/kubernetes/minikube/issues/2765)